Submodels/file_handling.py

```
@@ -0,0 +1,689 @@
+                                                                     bounds_error=True,
+                                                                     )  
+
+            # Construct output data
+            constructed_blade = {"chord_distribution": chord_distribution,
+                                 "sweep_distribution": sweep_distribution,
+                                 "thickness_distribution": thickness_distribution,
+                                 "camber_distribution": camber_distribution,
+                                 "entropy_distribution": entropy_distribution,
+                                 }
+            
+            return constructed_blade
```

Ruff (0.8.2)

Unnecessary assignment to `constructed_blade` before `return` statement

Remove unnecessary assignment

(RET504)